Doc/c-api/stable.rst

```diff
@@ -94,7 +94,15 @@ Stable ABI
 ----------
 
 To enable this, Python provides a *Stable ABI*: a set of symbols that will
-remain compatible across Python 3.x versions.
+remain ABI-compatible across Python 3.x versions.
+
+.. note::
+
+   The Stable ABI prevents ABI issues, like linker errors due to missing
+   symbols or data corruption due to changes in structure layouts or function
+   signatures.
+   However, other changes in Python can change the *behavior* of extensions.
```

Comment:
It might be worth adding a small example here (*e.g.*, reference counts, per the motivating issue).

Reply:
I don't think one example would do. Reference counts are less of a motivating issue and more of the straw that broke the camel's back.